main.go

```diff
@@ -90,12 +91,20 @@ func main() {
 	setupLog := ctrl.Log.WithName("setup")
 	klog.SetLogger(setupLog)
 
+	clientOpts := client.Options{
+		Cache: &client.CacheOptions{
+			DisableFor: getCacheExcludedObjectsTypes(),
+		},
+	}
+	metricsOpts := server.Options{
+		BindAddress: metricsAddr,
+	}
+
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+		Client:                 clientOpts,
 		Scheme:                 scheme,
-		MetricsBindAddress:     metricsAddr,
-		Port:                   9443,
```

Comment:
We don't need to set the port anymore?

Reply:
It's scaffolded deprecated configuration for webhook port which is not used in Image Controller.